application/org.openjdk.jmc.docs/html/GUID-3EC7E6C3-6A64-4659-B99E-CA675A9F6E0A.htm

```diff
@@ -79,6 +79,8 @@ <h1 id="JMCOH-GUID-3EC7E6C3-6A64-4659-B99E-CA675A9F6E0A" class="sect1"><span cla
 </li>
 <li><p>Make sure that there is a directory named <code>hsperfdata_<span class="codeinlineitalic">username</span></code> in the system's <code>tmp</code> directory, that it is writable by the user running JMC, and that the file system supports access control lists (ACLs).</p>
 </li>
+<li><p>Make sure you are you running a JDK >= 11. You can set the JDK bin directory either by&#160;<tt>PATH</tt>&#160;environment variable, or by adding&#160;<tt>-vm</tt>&#160;parameter to&#160;<tt>jmc.ini</tt>&#160;file (must be inserted before&#160;<tt>-vmargs</tt>&#160;section).</p>
```

Comment:
How about making explicit that it is Mission Control that should be running on the proposed version? "Make sure you are running Mission Control on "

Reply:
+1 done